docs/recovery.md

```diff
@@ -0,0 +1,351 @@
+```
+
+### Forced In-Place Rescan
+
+The recovery methods described above assume a clean slate for a node, so
+there's no existing UTXO or key data in the node's database. However, there're
+times when an _existing_ node may want to _manually_ rescan the chain. We have
+a tool for that! The tool is called
+[`dropwtxmgr`](https://github.com/btcsuite/btcwallet/tree/master/cmd/dropwtxmgr).
+It can be installed with the following command:
+```
+go get -v -u github.com/btcsuite/btcwallet/cmd/dropwtxmgr
+```
+
+The `dropwtxmgr` tool will _reset_ the best synced height of the wallet back to
+its birthday, or genesis if the birthday isn't known (for some older wallets).
+In order to run the tool, you must first **shutdown `lnd`**. Once `lnd` is
+shutdown, the rescan can be initiated with the following commands:
+```
+cp $HOME/.lnd/data/chain/bitcoin/mainnet/wallet.db $HOME/wallet.db # Copy the existing databse just in case!
+dropwtxmgr --db=$HOME/.lnd/data/chain/bitcoin/mainnet/wallet.db
+```
+
+Once the above command returns (if it hangs for a while, then `lnd` may not
+actually be shutdown, so double check!), `lnd` can be restarted. After it's
+restarted, then the wallet should being rescanning. An entry resembling the
+following will show up in the logs once it's complete:
+```
+[INF] LNWL: Finished rescan for 0 addresses (synced to block 3032830c812a4a6ea305d8ead13b52e9e69d6400ff3c997970b6f76fbc770920, height 748)
+```
+
+## Off-Chain Recovery
+
+After version `v0.6-beta` of `lnd`, the daemon now ships with a new feature
+called Static Channel Backups (SCBs). We call these _static_ as they only need
+to be obtained _once_: when the channel is created. From there on, a backup is
+good until the channel is closed. The backup contains all the information we
+need to initiate the Dataloss Protection (DLP) feature in the protocol, which
+ultimately leads to us recovering the funds from the channel _on-chain_. This
+is a foolproof _safe_
+
+We say _safe_, as care has been taken to ensure that there are no foot guns in
+this method of backing up channels, vs doing things like `rsync`ing or copying
+the `channel.db` file periodically. Those methods can be dangerous as one never
+knows if they have the latest state of a channel or not. Instead, we aim to
+provide a simple safe instead to allow users to recover the settled funds in
+their channels in the case of partial or complete data loss. The backups
+themselves are encrypted using a key derived from the user's seed, this way we
+protect privacy of the users channels in the back up state, and ensure that a
+random node can't attempt to import another user's channels.
+
+Given a valid SCB, the user will be able to recover _both_ funds that are fully
+settled within their channels. By "fully settled" we mean funds that are in the
+base commitment outputs, and not HTLCs.  We can only restore these funds as
+right after the channel is created, we have all the data required to make a
+backup. 
```

Comment:
additionally ", but lack information about the future HTLCs that the channel will process."